cloud/endagaweb/models.py

```diff
@@ -1460,6 +1463,26 @@ def create_ledger(sender, instance, created, **kwargs):
 post_save.connect(Network.create_billing_tiers, sender=Network)
 
 
+class NetworkDenomination(models.Model):
+    """Network has its own denomination bracket for rechange and validity
+
+    Subscriber status depends on recharge under denomination bracket
+    """
+    start_amount = models.BigIntegerField()
+    end_amount = models.BigIntegerField()
+    validity_days = models.PositiveIntegerField(blank=True, default=0)
+
+    # The denomination group associated with the network
+    network = models.ForeignKey('Network', null=True, on_delete=models.CASCADE)
+
+    def __unicode__(self):
+        return "Amount %s - %d  for %s(days)" % (
+            self.start_amount, self.end_amount, self.validity_days)
```

Comment:
please dont open new PRs for fixes to existing PRs its difficult to keep track of changes this way